pod.go

```diff
@@ -1163,9 +1163,9 @@ func togglePausePod(podID string, pause bool) (*Pod, error) {
 	return p, nil
 }
 
-func (p *Pod) attachDevices() error {
+func (p *Pod) attachDevices(create bool) error {
```

Comment:
Same comment here: `pod.state.State` should tell you about the pod being already created or not.